homeassistant/components/reisingerbridge/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+  "domain": "reisingerbridge",
+  "name": "Reisinger Bridge",
```

Comment:
Would Reisinger Drive not be more appropriate? I get that this integration communicates over a bridge, but the end functionality that this integration provides is the Reisinger Drive system right?

Reply:
Not quite, the component is part of a new product line that will be called this. But thanks for your idea